scripts/quick-start.sh

```diff
@@ -238,15 +238,15 @@ setup_worker_destination() {
         ${ROOT}/scripts/register-destination --name platform-cluster --context kind-platform $flags
     else
         cat "${ROOT}/config/samples/platform_v1alpha1_worker.yaml" | patch_statestore | kubectl --context kind-platform apply --filename -
-        install_gitops kind-worker worker-1
+        install_flux_gitops kind-worker worker-1
```

Comment:
Not sure you want to block this PR, but should we have an Argo flag for the worker gitops too?

Reply:
eventually yes!